### PR TITLE
Add Results list webcomponent

### PR DESCRIPTION
**Pull request by fgravin**
Add a web component to display a search results list.
It takes arguments:

- api-url
- layout (LIST,BLOCK,TEXT)
- lines _not covered yet_
- filters _not covered yet_

Exemple:

```html
<gn-results-list
  api-url="https://apps.titellus.net/geonetwork/srv/api"
  primary-color="{{primaryColor}}"
  layout="LIST"
  secondary-color="{{secondaryColor}}">
</gn-results-list>`,
```

### CODE REVIEW

**Comment by jahow** [comment]
Thanks for cleaning things up! the web component storybook (https://geonetwork.github.io/geonetwork-ui/results-list/storybook-wc) doesn't look like it's working. I'm testing locally.

File: webcomponents/gn-aggregated-records/src/app/gn-aggregated-records.module.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ export class GnAggregatedRecordsModule {
     const customButton = createCustomElement(GnAggregatedRecordsComponent, {
       injector: this.injector,
     })
-    customElements.define('gn-aggregated-records', customButton)
+    if (!customElements.get(WC_TAG_NAME)) {
```
Comment (jahow) [inline webcomponents/gn-aggregated-records/src/app/gn-aggregated-records.module.ts:31]: Maybe at some point we can factorize this in the `WcCommonModule`?

**Review by jahow — APPROVED** [review]
Awesome, thanks

**Comment by fgravin** [comment]
> Thanks for cleaning things up! the web component storybook (https://geonetwork.github.io/geonetwork-ui/results-list/storybook-wc) doesn't look like it's working. I'm testing locally.

Yes i have created an issue about this, tried many things but can't figure out yet.

**Pull request merged**